migrations/migrations/R__update-insert-complaint-from-staging.sql

```diff
@@ -376,23 +376,23 @@ OR REPLACE FUNCTION public.insert_complaint_from_staging (_complaint_identifier
     
     END IF;
    
-    UPDATE public.complaint 
+    UPDATE complaint.complaint com
```

Comment:
Few more specifications of schema, just noting them but I may be misunderstanding the need